lb_content_resolver/content_resolver.py

```diff
@@ -176,16 +176,18 @@ def add_file_to_index(self, relative_path, format, mtime):
         else:
             status = "added"
 
+        unknown_string = "[unknown]"
```

Comment:
This was undeclared in one of the formats it's usage was failing so I moved it up here to be common across all.